src/pages/Login/Login.js

```diff
@@ -1,7 +1,181 @@
 import React from 'react';
+import './Login.scss';
+// import './src/styles/variables.scss';
 
 function Login() {
-  return <p>Hi</p>;
+  return (
+    <body>
+      <div className="allWrap">
+        <div className="myPage">마이페이지</div>
+        <div className="customer">
+          <div className="newCustomer">
+            <div className="signupInput">
+              <p className="bold">신규 고객</p>
+              <div className="inputSignup">
+                <input type="text" placeholder="* 이름" />
+              </div>
+              <div className="inputSignup">
+                <input type="text" placeholder="* 이메일 주소" />
+              </div>
+              <div className="inputSignup">
+                <input type="password" placeholder="* 비밀번호" />
+              </div>
+              <label>
+                <input type="checkbox" value="showPw" /> 비밀번호 표시
+              </label>
+              <p className="red">
+                영문 대문자, 소문자, 숫자, 특수문자를 모두 조합하여 8자 이상으로
+                구성해주세요. (예시: Jomalone123!)
+              </p>
+            </div>
+            <div className="phoneNumber">
+              <p className="phoneNumberSize">*휴대전화 번호</p>
```

Comment:
```suggestion
            <div className="phoneNumberBox">
              <p className="phoneNumber">*휴대전화 번호</p>
```
여기도 이런식으로 관리하는 것이 직관적으로 보입니다!

다른 부분도 className을 변경해주세요!